lib/benchpark/experiment.py

```diff
@@ -70,12 +70,43 @@ def compute_modifiers_section(self):
 
     def compute_applications_section(self):
         # TODO: is there some reasonable default?
+        variables = {}
+        variables["n_ranks"] = num_procs  # TODO: num_procs will be defined in the child...
```

Comment:
If that is so, this will need to be `self.num_procs`, and that will be a new required method for all subclasses (which can be checked by `benchpark audit`, FWIW).